server/speakerList.json

```diff
@@ -1 +1 @@
-{"speakers":["bristech", "danfairs","michaellnorth","cloud8421","pimterry","samhogy","AlexEvade","FredAtBootstrap","shaundunne","SteffLocke","tojarrett","daveellender","sublimino","jezenthomas","SimonMoffatt"]}
+{"speakers":["bristech","danfairs","michaellnorth","cloud8421","pimterry","samhogy","AlexEvade","FredAtBootstrap","shaundunne","SteffLocke","tojarrett","daveellender","sublimino","jezenthomas","SimonMoffatt","d3v_rand0m"]}
```

Comment:
Is there a way to automatically avoid pushing updated speaker lists unless we really mean it?